GLTFSDK/Inc/GLTFSDK/BufferBuilder.h

```diff
@@ -84,6 +84,15 @@ namespace Microsoft
 
             ResourceWriter& GetResourceWriter();
             const ResourceWriter& GetResourceWriter() const;
+            bool setCurrentBufferURI ( const std::string &data )
```

Comment:
I think the missing semicolon is the main cause of the build break. Also, please use the same spacing style as the rest of the code.

```suggestion
            bool setCurrentBufferURI(const std::string &data);
```

Reply:
Thank you very much for the comments.
I think i fixed the code, alignment and spaces but it seems that the build still fails (on some files I did not modify, especially for the std::pow functions in the Math.h file). Do you have any suggestions for this issue? Or what is the next step ?
I apologize if this is taking longer than usual, but I am new to github procedures.

Reply:
I see a bunch of build errors, but they are all in BufferBuilder.h and .cpp. Have you built this locally?